templates/CRM/Admin/Form/Setting/Search.tpl

```diff
@@ -102,3 +110,15 @@
             <div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="bottom"}</div>
 
 </div>
+{literal}
+<script type="text/javascript">
+function handleCheckboxUpdate(event) {
```

Comment:
I think it is better to pass the name of the of the input field 'sortedQuicksearchOptions' instead of the name of the update handler, it will allow you to move this handleCheckboxUpdate function to the main template.

Reply:
The plan is to make it as generic as possible. You might not always want to update an input value when the sorting happens.

Reply:
>>> You might not always want to update an input value when the sorting happen

it is a sortable checkbox  list so I am not sure what else the one who reuse this would expect, I could be wrong of course but I think doing that will ensure that any one who reuse this won't need to re-write this logic over and over for each reuse.

Also Check my reply below, given that the $_POST already includes the correct order, we might not need this anymore.